test/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+  opensearch-node1:
+    image: opensearchproject/opensearch:latest
```

Comment:
We'll want a matrix of versions tested here. Can be added later, but consider it.

Reply:
Sure, will work on it.

Reply:
I've created an issue to keep track of it https://github.com/opensearch-project/opensearch-api-specification/issues/43.